<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlslEditor</class>
 <widget class="QWidget" name="GlslEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GLSL-Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QComboBox" name="shaderComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="restorePushButton">
     <property name="text">
      <string>&amp;Restore</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="applyPushButton">
     <property name="text">
      <string>&amp;Apply</string>
     </property>
     <property name="shortcut">
      <string>F5</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="GlslTextEdit" name="glslTextEdit">
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GlslTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>gui/glsltextedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>shaderComboBox</tabstop>
  <tabstop>glslTextEdit</tabstop>
  <tabstop>restorePushButton</tabstop>
  <tabstop>applyPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
